habana/tensorflow/training/docker/2.9/py3/synapseai1.5.0/Dockerfile.hpu

```diff
@@ -0,0 +1,120 @@
+FROM vault.habana.ai/gaudi-docker/1.5.0/ubuntu20.04/habanalabs/tensorflow-installer-tf-cpu-2.9.1:latest
```

Comment:
I think we haven't added the buildspec for the older synapseAI (1.4.1) version yet. Can those be added please?

Reply:
do you mean synapseAI(1.4.1) for TF2.9?

Reply:
I meant `buildspec-1-10-syai-1-4.yml` for PT and `buildspec-2-8-syai-1-4.yml` for TF. (The copies of the buildspec files that we retain in our repo)

Reply:
done